dev/content/pages/About.md

```diff
@@ -105,15 +105,18 @@ save_as: index.html
 <!-- Left -->
 <div class="about-box">
 <h1 style="margin-top:0; text-align: left;">About Me</h1>
-    <p>I am a data scientist at Google where I use data to inform decisions, although all my opinions here are my own. I have previously worked at Sweetgreen, designing systems that would benefit team members and communities through sustainable and healthy food, and SpaceX, creating tools that would ultimately launch rocket ships. I feel fortunate to have worked with amazing teams throughout my career.</p>
-    <p>There are 3 things I strive to be:</p>
+    <p>I am a data scientist at Google. My focus is supporting strategic decision making with data, although all my opinions here are my own. Previously worked at Sweetgreen and SpaceX, doing the same, using data to inform staffing strategy and rocket launch cadence.</p>
+    
+    <p>To become a better "decision" scientist I'm currently focusing on three topics</p>
     <ol>
-        <li>An expert computational mathematician</li>
-        <li>An effective community member and leader</li>
-        <li>A knowledge contributor</li>
+        <li>Organizational Influence and Leadership</li>
+        <li>Advanced applied mathematics e.g. Causal Inference, Decistion Theory</li>
+        <li>Business Domain expertise</li>
```

Comment:
1. The capitalization seemed kinda inconsistent, so I lower-cased everything if the word wasn't the starting one.
2. Personally, I feel a little iffy stating "business domain expertise" in a non-work/personal setting for two reasons: First, I think of business/domain expertise as company-specific or confidential. Second, usually you need to be an expert in a specific field and you are not stating any field. Could it be more like "business applications/case studies of businesses using statistics to drive a decision", or "domain expertise in <SOMETHING>"? I feel like you will have an easier time going with the former option.
```suggestion
        <li>Organizational influence and leadership</li>
        <li>Advanced applied mathematics, e.g. causal Inference, decision theory</li>
        <li>Business domain expertise</li>
```

Reply:
Agree I'll change it